"[BeforeEach] [sig-network] Services
  /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:174
STEP:
Creating a kubernetes client
Mar 14 00:10:11.277: INFO: >>> kubeConfig:
/tmp/kubeconfig-458657413
STEP: Building a namespace api object, basename
services
STEP: Waiting for a default service account to be provisioned
in namespace
[BeforeEach] [sig-network] Services
  /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/service.go:745
[It]
should be able to change the type from ExternalName to NodePort [Conformance]

\ /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:629
STEP:
creating a service externalname-service with the type=ExternalName in namespace
services-6203
STEP: changing the ExternalName service to type=NodePort
STEP:
creating replication controller externalname-service in namespace services-6203
Mar
14 00:10:23.435: INFO: Creating new exec pod
Mar 14 00:10:32.458: INFO:
Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203
exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80'
Mar
14 00:10:33.764: INFO: rc: 1
Mar 14 00:10:33.764: INFO: Service reachability
failing with error: error running /usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80:
Command stdout:

stderr:
+ nc -zv -t -w 2
externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:34.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:35.116: INFO: rc: 1
Mar 14 00:10:35.116:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:35.768: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:36.214: INFO: rc: 1
Mar 14 00:10:36.214:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:36.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:37.075: INFO: rc: 1
Mar 14 00:10:37.075:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:37.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:38.194: INFO: rc: 1
Mar 14 00:10:38.194:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:38.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:39.106: INFO: rc: 1
Mar 14 00:10:39.106:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:39.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:40.184: INFO: rc: 1
Mar 14 00:10:40.184:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:40.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:41.070: INFO: rc: 1
Mar 14 00:10:41.070:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:41.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:42.579: INFO: rc: 1
Mar 14 00:10:42.579:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:42.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:43.070: INFO: rc: 1
Mar 14 00:10:43.070:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:43.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:44.203: INFO: rc: 1
Mar 14 00:10:44.203:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:44.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:45.354: INFO: rc: 1
Mar 14 00:10:45.354:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:45.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:46.099: INFO: rc: 1
Mar 14 00:10:46.099:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:46.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:47.076: INFO: rc: 1
Mar 14 00:10:47.076:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:47.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:48.097: INFO: rc: 1
Mar 14 00:10:48.097:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:48.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:49.094: INFO: rc: 1
Mar 14 00:10:49.094:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:49.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:50.130: INFO: rc: 1
Mar 14 00:10:50.130:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:50.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:51.075: INFO: rc: 1
Mar 14 00:10:51.075:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:51.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:52.096: INFO: rc: 1
Mar 14 00:10:52.096:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:52.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:53.088: INFO: rc: 1
Mar 14 00:10:53.088:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:53.768: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:54.114: INFO: rc: 1
Mar 14 00:10:54.114:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:54.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:55.615: INFO: rc: 1
Mar 14 00:10:55.615:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:55.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:56.110: INFO: rc: 1
Mar 14 00:10:56.110:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:56.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:57.228: INFO: rc: 1
Mar 14 00:10:57.228:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:57.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:58.091: INFO: rc: 1
Mar 14 00:10:58.092:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:58.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:10:59.319: INFO: rc: 1
Mar 14 00:10:59.320:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:10:59.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:00.127: INFO: rc: 1
Mar 14 00:11:00.127:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:00.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:01.082: INFO: rc: 1
Mar 14 00:11:01.082:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:01.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:02.333: INFO: rc: 1
Mar 14 00:11:02.333:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:02.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:03.818: INFO: rc: 1
Mar 14 00:11:03.818:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:04.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:05.355: INFO: rc: 1
Mar 14 00:11:05.355:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:05.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:06.381: INFO: rc: 1
Mar 14 00:11:06.381:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:06.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:07.242: INFO: rc: 1
Mar 14 00:11:07.242:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:07.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:08.854: INFO: rc: 1
Mar 14 00:11:08.854:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:09.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:10.199: INFO: rc: 1
Mar 14 00:11:10.199:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:10.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:11.081: INFO: rc: 1
Mar 14 00:11:11.081:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:11.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:12.182: INFO: rc: 1
Mar 14 00:11:12.182:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:12.772: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:13.315: INFO: rc: 1
Mar 14 00:11:13.315:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:13.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:14.735: INFO: rc: 1
Mar 14 00:11:14.735:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:14.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:15.118: INFO: rc: 1
Mar 14 00:11:15.118:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:15.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:16.215: INFO: rc: 1
Mar 14 00:11:16.215:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:16.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:17.361: INFO: rc: 1
Mar 14 00:11:17.361:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:17.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:18.234: INFO: rc: 1
Mar 14 00:11:18.234:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:18.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:19.204: INFO: rc: 1
Mar 14 00:11:19.204:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:19.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:20.183: INFO: rc: 1
Mar 14 00:11:20.183:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:20.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:21.351: INFO: rc: 1
Mar 14 00:11:21.351:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:21.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:22.169: INFO: rc: 1
Mar 14 00:11:22.169:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:22.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:23.154: INFO: rc: 1
Mar 14 00:11:23.154:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:23.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:24.176: INFO: rc: 1
Mar 14 00:11:24.176:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:24.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:25.110: INFO: rc: 1
Mar 14 00:11:25.110:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:25.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:26.168: INFO: rc: 1
Mar 14 00:11:26.168:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:26.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:27.448: INFO: rc: 1
Mar 14 00:11:27.448:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:27.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:28.115: INFO: rc: 1
Mar 14 00:11:28.115:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:28.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:29.136: INFO: rc: 1
Mar 14 00:11:29.136:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:29.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:30.310: INFO: rc: 1
Mar 14 00:11:30.310:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:30.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:31.931: INFO: rc: 1
Mar 14 00:11:31.931:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:32.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:33.113: INFO: rc: 1
Mar 14 00:11:33.113:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:33.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:34.125: INFO: rc: 1
Mar 14 00:11:34.125:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:34.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:35.382: INFO: rc: 1
Mar 14 00:11:35.382:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:35.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:36.117: INFO: rc: 1
Mar 14 00:11:36.117:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:36.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:37.104: INFO: rc: 1
Mar 14 00:11:37.105:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:37.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:38.128: INFO: rc: 1
Mar 14 00:11:38.128:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:38.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:39.166: INFO: rc: 1
Mar 14 00:11:39.166:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:39.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:40.132: INFO: rc: 1
Mar 14 00:11:40.132:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:40.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:41.124: INFO: rc: 1
Mar 14 00:11:41.124:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:41.769: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:42.134: INFO: rc: 1
Mar 14 00:11:42.134:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:42.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:43.164: INFO: rc: 1
Mar 14 00:11:43.164:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:43.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:44.152: INFO: rc: 1
Mar 14 00:11:44.152:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:44.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:45.106: INFO: rc: 1
Mar 14 00:11:45.106:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:45.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:46.327: INFO: rc: 1
Mar 14 00:11:46.327:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:46.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:47.138: INFO: rc: 1
Mar 14 00:11:47.139:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:47.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:48.740: INFO: rc: 1
Mar 14 00:11:48.741:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:48.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:49.118: INFO: rc: 1
Mar 14 00:11:49.118:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:49.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:50.237: INFO: rc: 1
Mar 14 00:11:50.237:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:50.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:51.140: INFO: rc: 1
Mar 14 00:11:51.140:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:51.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:52.266: INFO: rc: 1
Mar 14 00:11:52.267:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:52.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:53.171: INFO: rc: 1
Mar 14 00:11:53.171:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:53.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:54.303: INFO: rc: 1
Mar 14 00:11:54.303:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:54.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:55.095: INFO: rc: 1
Mar 14 00:11:55.096:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:55.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:56.314: INFO: rc: 1
Mar 14 00:11:56.315:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:56.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:57.125: INFO: rc: 1
Mar 14 00:11:57.125:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:57.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:58.672: INFO: rc: 1
Mar 14 00:11:58.672:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:58.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:11:59.144: INFO: rc: 1
Mar 14 00:11:59.144:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:11:59.768: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:00.189: INFO: rc: 1
Mar 14 00:12:00.189:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:00.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:01.141: INFO: rc: 1
Mar 14 00:12:01.141:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:01.769: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:02.164: INFO: rc: 1
Mar 14 00:12:02.164:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:02.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:03.129: INFO: rc: 1
Mar 14 00:12:03.129:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:03.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:04.132: INFO: rc: 1
Mar 14 00:12:04.132:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:04.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:06.150: INFO: rc: 1
Mar 14 00:12:06.150:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:06.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:07.203: INFO: rc: 1
Mar 14 00:12:07.203:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:07.768: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:08.430: INFO: rc: 1
Mar 14 00:12:08.430:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:08.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:09.186: INFO: rc: 1
Mar 14 00:12:09.186:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:09.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:10.775: INFO: rc: 1
Mar 14 00:12:10.775:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:11.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:12.446: INFO: rc: 1
Mar 14 00:12:12.446:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:12.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:13.562: INFO: rc: 1
Mar 14 00:12:13.563:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:13.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:14.147: INFO: rc: 1
Mar 14 00:12:14.147:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:14.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:15.357: INFO: rc: 1
Mar 14 00:12:15.357:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:15.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:16.135: INFO: rc: 1
Mar 14 00:12:16.135:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:16.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:17.119: INFO: rc: 1
Mar 14 00:12:17.119:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:17.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:18.197: INFO: rc: 1
Mar 14 00:12:18.197:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:18.766: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:19.139: INFO: rc: 1
Mar 14 00:12:19.139:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:19.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:20.621: INFO: rc: 1
Mar 14 00:12:20.621:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:20.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:21.127: INFO: rc: 1
Mar 14 00:12:21.127:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:21.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:22.212: INFO: rc: 1
Mar 14 00:12:22.212:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:22.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:23.177: INFO: rc: 1
Mar 14 00:12:23.177:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:23.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:24.775: INFO: rc: 1
Mar 14 00:12:24.775:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:25.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:26.178: INFO: rc: 1
Mar 14 00:12:26.178:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:26.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:27.165: INFO: rc: 1
Mar 14 00:12:27.165:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:27.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:28.284: INFO: rc: 1
Mar 14 00:12:28.284:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:28.764: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:29.171: INFO: rc: 1
Mar 14 00:12:29.171:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:29.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:30.657: INFO: rc: 1
Mar 14 00:12:30.657:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:30.767: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:31.183: INFO: rc: 1
Mar 14 00:12:31.183:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:31.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:32.328: INFO: rc: 1
Mar 14 00:12:32.328:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:32.772: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:33.639: INFO: rc: 1
Mar 14 00:12:33.639:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:33.765: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:34.219: INFO: rc: 1
Mar 14 00:12:34.219:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:34.219: INFO: Running '/usr/local/bin/kubectl --kubeconfig=/tmp/kubeconfig-458657413
--namespace=services-6203 exec execpodq6fvf -- /bin/sh -x -c nc -zv -t -w
2 externalname-service 80'
Mar 14 00:12:34.674: INFO: rc: 1
Mar 14 00:12:34.674:
INFO: Service reachability failing with error: error running /usr/local/bin/kubectl
--kubeconfig=/tmp/kubeconfig-458657413 --namespace=services-6203 exec execpodq6fvf
-- /bin/sh -x -c nc -zv -t -w 2 externalname-service 80:
Command stdout:

stderr:
+
nc -zv -t -w 2 externalname-service 80
nc: getaddrinfo: Name does not resolve
command
terminated with exit code 1

error:
exit status 1
Retrying...
Mar 14
00:12:34.674: FAIL: Unexpected error:
    <*errors.errorString | 0xc001050270>:
{
        s: \"service is not reachable within 2m0s timeout on endpoint
externalname-service:80 over TCP protocol\",
    }
    service is not
reachable within 2m0s timeout on endpoint externalname-service:80 over TCP
protocol
occurred

Full Stack Trace
k8s.io/kubernetes/test/e2e/network.glob..func24.15()
\t/workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/service.go:1701
+0x358
k8s.io/kubernetes/test/e2e.RunE2ETests(0xc0019de180)
\t_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/e2e.go:130
+0x36c
k8s.io/kubernetes/test/e2e.TestE2E(0xc0019de180)
\t_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/e2e_test.go:144
+0x2b
testing.tRunner(0xc0019de180, 0x4fa8cc8)
\t/usr/local/go/src/testing/testing.go:1123
+0xef
created by testing.(*T).Run
\t/usr/local/go/src/testing/testing.go:1168
+0x2b3
Mar 14 00:12:34.675: INFO: Cleaning up the ExternalName to NodePort
test service
[AfterEach] [sig-network] Services
  /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:175
STEP:
Collecting events from namespace \"services-6203\".
STEP: Found 19 events.
Mar
14 00:12:34.725: INFO: At 0001-01-01 00:00:00 +0000 UTC - event for execpodq6fvf:
{ } Scheduled: Successfully assigned services-6203/execpodq6fvf to monk
Mar
14 00:12:34.725: INFO: At 0001-01-01 00:00:00 +0000 UTC - event for externalname-service-8l5sf:
{ } Scheduled: Successfully assigned services-6203/externalname-service-8l5sf
to ham
Mar 14 00:12:34.725: INFO: At 0001-01-01 00:00:00 +0000 UTC - event
for externalname-service-wlp8x: { } Scheduled: Successfully assigned services-6203/externalname-service-wlp8x
to doc
Mar 14 00:12:34.725: INFO: At 2021-03-14 00:10:11 +0000 UTC - event
for externalname-service: {replication-controller } SuccessfulCreate: Created
pod: externalname-service-wlp8x
Mar 14 00:12:34.725: INFO: At 2021-03-14
00:10:11 +0000 UTC - event for externalname-service: {replication-controller
} SuccessfulCreate: Created pod: externalname-service-8l5sf
Mar 14 00:12:34.725:
INFO: At 2021-03-14 00:10:11 +0000 UTC - event for externalname-service-8l5sf:
{kubelet ham} DNSConfigForming: Nameserver limits were exceeded, some nameservers
have been omitted, the applied nameserver line is: 2001:558:feed::1 2001:558:feed::2
75.75.75.75
Mar 14 00:12:34.726: INFO: At 2021-03-14 00:10:11 +0000 UTC
- event for externalname-service-wlp8x: {kubelet doc} DNSConfigForming:
Nameserver limits were exceeded, some nameservers have been omitted, the
applied nameserver line is: 75.75.75.75 75.75.76.76 2001:558:feed::1
Mar
14 00:12:34.726: INFO: At 2021-03-14 00:10:12 +0000 UTC - event for externalname-service-8l5sf:
{kubelet ham} Pulling: Pulling image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
Mar
14 00:12:34.726: INFO: At 2021-03-14 00:10:12 +0000 UTC - event for externalname-service-wlp8x:
{kubelet doc} Pulling: Pulling image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
Mar
14 00:12:34.726: INFO: At 2021-03-14 00:10:18 +0000 UTC - event for externalname-service-8l5sf:
{kubelet ham} Pulled: Successfully pulled image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
in 5.293577169s
Mar 14 00:12:34.726: INFO: At 2021-03-14 00:10:18 +0000
UTC - event for externalname-service-8l5sf: {kubelet ham} Started: Started
container externalname-service
Mar 14 00:12:34.726: INFO: At 2021-03-14
00:10:18 +0000 UTC - event for externalname-service-8l5sf: {kubelet ham}
Created: Created container externalname-service
Mar 14 00:12:34.726: INFO:
At 2021-03-14 00:10:21 +0000 UTC - event for externalname-service-wlp8x:
{kubelet doc} Pulled: Successfully pulled image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
in 8.247771854s
Mar 14 00:12:34.726: INFO: At 2021-03-14 00:10:21 +0000
UTC - event for externalname-service-wlp8x: {kubelet doc} Created: Created
container externalname-service
Mar 14 00:12:34.726: INFO: At 2021-03-14
00:10:21 +0000 UTC - event for externalname-service-wlp8x: {kubelet doc}
Started: Started container externalname-service
Mar 14 00:12:34.726: INFO:
At 2021-03-14 00:10:24 +0000 UTC - event for execpodq6fvf: {kubelet monk}
Pulling: Pulling image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
Mar
14 00:12:34.726: INFO: At 2021-03-14 00:10:28 +0000 UTC - event for execpodq6fvf:
{kubelet monk} Pulled: Successfully pulled image \"k8s.gcr.io/e2e-test-images/agnhost:2.21\"
in 4.577653405s
Mar 14 00:12:34.726: INFO: At 2021-03-14 00:10:29 +0000
UTC - event for execpodq6fvf: {kubelet monk} Created: Created container
agnhost-container
Mar 14 00:12:34.726: INFO: At 2021-03-14 00:10:29 +0000
UTC - event for execpodq6fvf: {kubelet monk} Started: Started container
agnhost-container
Mar 14 00:12:34.728: INFO: POD                         NODE
\ PHASE    GRACE  CONDITIONS
Mar 14 00:12:34.728: INFO: execpodq6fvf                monk
\ Running         [{Initialized True 0001-01-01 00:00:00 +0000 UTC 2021-03-14
00:10:23 +0000 UTC  } {Ready True 0001-01-01 00:00:00 +0000 UTC 2021-03-14
00:10:29 +0000 UTC  } {ContainersReady True 0001-01-01 00:00:00 +0000 UTC
2021-03-14 00:10:29 +0000 UTC  } {PodScheduled True 0001-01-01 00:00:00
+0000 UTC 2021-03-14 00:10:23 +0000 UTC  }]
Mar 14 00:12:34.728: INFO:
externalname-service-8l5sf  ham   Running         [{Initialized True 0001-01-01
00:00:00 +0000 UTC 2021-03-14 00:10:11 +0000 UTC  } {Ready True 0001-01-01
00:00:00 +0000 UTC 2021-03-14 00:10:19 +0000 UTC  } {ContainersReady True
0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:19 +0000 UTC  } {PodScheduled
True 0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:11 +0000 UTC  }]
Mar
14 00:12:34.728: INFO: externalname-service-wlp8x  doc   Running         [{Initialized
True 0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:11 +0000 UTC  } {Ready
True 0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:22 +0000 UTC  } {ContainersReady
True 0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:22 +0000 UTC  } {PodScheduled
True 0001-01-01 00:00:00 +0000 UTC 2021-03-14 00:10:11 +0000 UTC  }]
Mar
14 00:12:34.728: INFO: 
Mar 14 00:12:34.730: INFO: 
Logging node info
for node doc
Mar 14 00:12:34.732: INFO: Node Info: &Node{ObjectMeta:{doc
\   fe7dd4c5-54f2-4510-9f74-ab7ba7d49f48 1806 0 2021-03-13 23:58:55 +0000
UTC <nil> <nil> map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/os:linux
kubernetes.io/arch:amd64 kubernetes.io/hostname:doc kubernetes.io/os:linux
node-role.kubernetes.io/controller: node-role.kubernetes.io/worker:] map[node.alpha.kubernetes.io/ttl:0
projectcalico.org/IPv4Address:192.168.0.242/24 projectcalico.org/IPv4IPIPTunnelAddr:10.200.127.192
volumes.kubernetes.io/controller-managed-attach-detach:true] [] []  [{kube-controller-manager
Update v1 2021-03-13 23:58:55 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\"f:node.alpha.kubernetes.io/ttl\":{}}},\"f:spec\":{\"f:podCIDR\":{},\"f:podCIDRs\":{\".\":{},\"v:\\\"10.200.0.0/24\\\"\":{}}}}}
{kubectl Update v1 2021-03-13 23:58:56 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:labels\":{\"f:node-role.kubernetes.io/controller\":{},\"f:node-role.kubernetes.io/worker\":{}}}}}
{kubelet Update v1 2021-03-14 00:04:30 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\".\":{},\"f:volumes.kubernetes.io/controller-managed-attach-detach\":{}},\"f:labels\":{\".\":{},\"f:beta.kubernetes.io/arch\":{},\"f:beta.kubernetes.io/os\":{},\"f:kubernetes.io/arch\":{},\"f:kubernetes.io/hostname\":{},\"f:kubernetes.io/os\":{}}},\"f:status\":{\"f:addresses\":{\".\":{},\"k:{\\\"type\\\":\\\"Hostname\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"InternalIP\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}}},\"f:allocatable\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:capacity\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:conditions\":{\".\":{},\"k:{\\\"type\\\":\\\"DiskPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"MemoryPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"PIDPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"Ready\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}},\"f:daemonEndpoints\":{\"f:kubeletEndpoint\":{\"f:Port\":{}}},\"f:images\":{},\"f:nodeInfo\":{\"f:architecture\":{},\"f:bootID\":{},\"f:containerRuntimeVersion\":{},\"f:kernelVersion\":{},\"f:kubeProxyVersion\":{},\"f:kubeletVersion\":{},\"f:machineID\":{},\"f:operatingSystem\":{},\"f:osImage\":{},\"f:systemUUID\":{}}}}}
{calico-node Update v1 2021-03-14 00:04:34 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\"f:projectcalico.org/IPv4Address\":{},\"f:projectcalico.org/IPv4IPIPTunnelAddr\":{}}},\"f:status\":{\"f:conditions\":{\"k:{\\\"type\\\":\\\"NetworkUnavailable\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}}}}}]},Spec:NodeSpec{PodCIDR:10.200.0.0/24,DoNotUseExternalID:,ProviderID:,Unschedulable:false,Taints:[]Taint{},ConfigSource:nil,PodCIDRs:[10.200.0.0/24],},Status:NodeStatus{Capacity:ResourceList{cpu:
{{4 0} {<nil>} 4 DecimalSI},ephemeral-storage: {{17785946112 0} {<nil>}
16962Mi BinarySI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory: {{8145018880
0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Allocatable:ResourceList{cpu:
{{4 0} {<nil>} 4 DecimalSI},ephemeral-storage: {{16007351475 0} {<nil>}
16007351475 DecimalSI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory:
{{8040161280 0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Phase:,Conditions:[]NodeCondition{NodeCondition{Type:NetworkUnavailable,Status:False,LastHeartbeatTime:2021-03-14
00:04:34 +0000 UTC,LastTransitionTime:2021-03-14 00:04:34 +0000 UTC,Reason:CalicoIsUp,Message:Calico
is running on this node,},NodeCondition{Type:MemoryPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:32 +0000 UTC,LastTransitionTime:2021-03-13 23:58:55 +0000 UTC,Reason:KubeletHasSufficientMemory,Message:kubelet
has sufficient memory available,},NodeCondition{Type:DiskPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:32 +0000 UTC,LastTransitionTime:2021-03-13 23:58:55 +0000 UTC,Reason:KubeletHasNoDiskPressure,Message:kubelet
has no disk pressure,},NodeCondition{Type:PIDPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:32 +0000 UTC,LastTransitionTime:2021-03-13 23:58:55 +0000 UTC,Reason:KubeletHasSufficientPID,Message:kubelet
has sufficient PID available,},NodeCondition{Type:Ready,Status:True,LastHeartbeatTime:2021-03-14
00:10:32 +0000 UTC,LastTransitionTime:2021-03-13 23:59:05 +0000 UTC,Reason:KubeletReady,Message:kubelet
is posting ready status,},},Addresses:[]NodeAddress{NodeAddress{Type:InternalIP,Address:192.168.0.242,},NodeAddress{Type:Hostname,Address:doc,},},DaemonEndpoints:NodeDaemonEndpoints{KubeletEndpoint:DaemonEndpoint{Port:10250,},},NodeInfo:NodeSystemInfo{MachineID:cf27580027864897b10943e217a56fc8,SystemUUID:e2279db7-eb4e-d849-94c3-02b73b9bc25d,BootID:28d2ce86-9194-44df-b866-49fc1fe49544,KernelVersion:4.18.0-240.10.1.el8_3.x86_64,OSImage:CentOS
Linux 8,ContainerRuntimeVersion:containerd://1.4.3,KubeletVersion:v1.20.1,KubeProxyVersion:v1.20.1,OperatingSystem:linux,Architecture:amd64,},Images:[]ContainerImage{ContainerImage{Names:[k8s.gcr.io/conformance@sha256:9f165f5d6393246eae4ea77cfc99d7f39b7475bd7650c72759d44f1f2cd3bafe
k8s.gcr.io/conformance:v1.20.1],SizeBytes:70202323,},ContainerImage{Names:[docker.io/calico/node@sha256:e207db848adb688ae3edac439c650b0ec44c0675efc3b735721f7541e0320100
docker.io/calico/node:v3.18.0],SizeBytes:60502888,},ContainerImage{Names:[k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
k8s.gcr.io/e2e-test-images/agnhost:2.21],SizeBytes:46251468,},ContainerImage{Names:[docker.io/calico/cni@sha256:9d692b3ce9003469f5d50de91dc66ee37d76d78715ed5c1f4884b5d901411489
docker.io/calico/cni:v3.18.0],SizeBytes:44689478,},ContainerImage{Names:[projects.registry.vmware.com/sonobuoy/sonobuoy@sha256:ba5c473283dbadcbc56be1149ae45d60df020260ba851e3fc28acc445b300a54
projects.registry.vmware.com/sonobuoy/sonobuoy:v0.20.0],SizeBytes:36608089,},ContainerImage{Names:[docker.io/calico/kube-controllers@sha256:12515fb6e2f35a2ce86133cb2f17fa8039381c9ac031c2b300af398f8589651f
docker.io/calico/kube-controllers:v3.18.0],SizeBytes:22593186,},ContainerImage{Names:[docker.io/coredns/coredns@sha256:cc8fb77bc2a0541949d1d9320a641b82fd392b0d3d8145469ca4709ae769980e
docker.io/coredns/coredns:1.8.0],SizeBytes:12945155,},ContainerImage{Names:[docker.io/calico/pod2daemon-flexvol@sha256:63939cfbd430345c6add6548fc7a22e3082d0738f455ae747dd6264c834bcd4e
docker.io/calico/pod2daemon-flexvol:v3.18.0],SizeBytes:9327966,},ContainerImage{Names:[k8s.gcr.io/pause@sha256:927d98197ec1141a368550822d18fa1c60bdae27b78b0c004f705f548c07814f
k8s.gcr.io/pause:3.2],SizeBytes:299513,},},VolumesInUse:[],VolumesAttached:[]AttachedVolume{},Config:nil,},}
Mar
14 00:12:34.732: INFO: 
Logging kubelet events for node doc
Mar 14 00:12:34.735:
INFO: 
Logging pods the kubelet thinks is on node doc
Mar 14 00:12:34.754:
INFO: calico-kube-controllers-6949477b58-6j4sl started at 2021-03-14 00:04:01
+0000 UTC (0+1 container statuses recorded)
Mar 14 00:12:34.754: INFO:
\tContainer calico-kube-controllers ready: true, restart count 0
Mar 14
00:12:34.754: INFO: calico-node-2lc9g started at 2021-03-14 00:04:02 +0000
UTC (3+1 container statuses recorded)
Mar 14 00:12:34.754: INFO: \tInit
container upgrade-ipam ready: true, restart count 0
Mar 14 00:12:34.754:
INFO: \tInit container install-cni ready: true, restart count 0
Mar 14
00:12:34.754: INFO: \tInit container flexvol-driver ready: true, restart
count 0
Mar 14 00:12:34.754: INFO: \tContainer calico-node ready: true,
restart count 0
Mar 14 00:12:34.754: INFO: coredns-75c4c67f76-5pdhw started
at 2021-03-14 00:04:03 +0000 UTC (0+1 container statuses recorded)
Mar
14 00:12:34.754: INFO: \tContainer coredns ready: true, restart count 0
Mar
14 00:12:34.754: INFO: sonobuoy started at 2021-03-14 00:09:00 +0000 UTC
(0+1 container statuses recorded)
Mar 14 00:12:34.754: INFO: \tContainer
kube-sonobuoy ready: true, restart count 0
Mar 14 00:12:34.754: INFO: sonobuoy-e2e-job-49f59f66bab74873
started at 2021-03-14 00:09:58 +0000 UTC (0+2 container statuses recorded)
Mar
14 00:12:34.754: INFO: \tContainer e2e ready: true, restart count 0
Mar
14 00:12:34.754: INFO: \tContainer sonobuoy-worker ready: true, restart
count 0
Mar 14 00:12:34.754: INFO: externalname-service-wlp8x started at
2021-03-14 00:10:11 +0000 UTC (0+1 container statuses recorded)
Mar 14
00:12:34.754: INFO: \tContainer externalname-service ready: true, restart
count 0
Mar 14 00:12:35.324: INFO: 
Latency metrics for node doc
Mar
14 00:12:35.324: INFO: 
Logging node info for node ham
Mar 14 00:12:35.332:
INFO: Node Info: &Node{ObjectMeta:{ham    1b2313c1-9075-4dff-92fd-632ba89c8488
1788 0 2021-03-14 00:01:20 +0000 UTC <nil> <nil> map[beta.kubernetes.io/arch:amd64
beta.kubernetes.io/os:linux kubernetes.io/arch:amd64 kubernetes.io/hostname:ham
kubernetes.io/os:linux node-role.kubernetes.io/worker:] map[node.alpha.kubernetes.io/ttl:0
projectcalico.org/IPv4Address:192.168.0.243/24 projectcalico.org/IPv4IPIPTunnelAddr:10.200.224.64
volumes.kubernetes.io/controller-managed-attach-detach:true] [] []  [{kubectl
Update v1 2021-03-14 00:01:21 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:labels\":{\"f:node-role.kubernetes.io/worker\":{}}}}}
{kube-controller-manager Update v1 2021-03-14 00:01:22 +0000 UTC FieldsV1
{\"f:metadata\":{\"f:annotations\":{\"f:node.alpha.kubernetes.io/ttl\":{}}},\"f:spec\":{\"f:podCIDR\":{},\"f:podCIDRs\":{\".\":{},\"v:\\\"10.200.1.0/24\\\"\":{}}}}}
{calico-node Update v1 2021-03-14 00:04:20 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\"f:projectcalico.org/IPv4Address\":{},\"f:projectcalico.org/IPv4IPIPTunnelAddr\":{}}},\"f:status\":{\"f:conditions\":{\"k:{\\\"type\\\":\\\"NetworkUnavailable\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}}}}}
{kubelet Update v1 2021-03-14 00:04:22 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\".\":{},\"f:volumes.kubernetes.io/controller-managed-attach-detach\":{}},\"f:labels\":{\".\":{},\"f:beta.kubernetes.io/arch\":{},\"f:beta.kubernetes.io/os\":{},\"f:kubernetes.io/arch\":{},\"f:kubernetes.io/hostname\":{},\"f:kubernetes.io/os\":{}}},\"f:status\":{\"f:addresses\":{\".\":{},\"k:{\\\"type\\\":\\\"Hostname\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"InternalIP\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}}},\"f:allocatable\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:capacity\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:conditions\":{\".\":{},\"k:{\\\"type\\\":\\\"DiskPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"MemoryPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"PIDPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"Ready\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}},\"f:daemonEndpoints\":{\"f:kubeletEndpoint\":{\"f:Port\":{}}},\"f:images\":{},\"f:nodeInfo\":{\"f:architecture\":{},\"f:bootID\":{},\"f:containerRuntimeVersion\":{},\"f:kernelVersion\":{},\"f:kubeProxyVersion\":{},\"f:kubeletVersion\":{},\"f:machineID\":{},\"f:operatingSystem\":{},\"f:osImage\":{},\"f:systemUUID\":{}}}}}]},Spec:NodeSpec{PodCIDR:10.200.1.0/24,DoNotUseExternalID:,ProviderID:,Unschedulable:false,Taints:[]Taint{},ConfigSource:nil,PodCIDRs:[10.200.1.0/24],},Status:NodeStatus{Capacity:ResourceList{cpu:
{{2 0} {<nil>} 2 DecimalSI},ephemeral-storage: {{17785946112 0} {<nil>}
16962Mi BinarySI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory: {{8145461248
0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Allocatable:ResourceList{cpu:
{{2 0} {<nil>} 2 DecimalSI},ephemeral-storage: {{16007351475 0} {<nil>}
16007351475 DecimalSI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory:
{{8040603648 0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Phase:,Conditions:[]NodeCondition{NodeCondition{Type:NetworkUnavailable,Status:False,LastHeartbeatTime:2021-03-14
00:04:19 +0000 UTC,LastTransitionTime:2021-03-14 00:04:19 +0000 UTC,Reason:CalicoIsUp,Message:Calico
is running on this node,},NodeCondition{Type:MemoryPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:24 +0000 UTC,LastTransitionTime:2021-03-14 00:01:20 +0000 UTC,Reason:KubeletHasSufficientMemory,Message:kubelet
has sufficient memory available,},NodeCondition{Type:DiskPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:24 +0000 UTC,LastTransitionTime:2021-03-14 00:01:20 +0000 UTC,Reason:KubeletHasNoDiskPressure,Message:kubelet
has no disk pressure,},NodeCondition{Type:PIDPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:24 +0000 UTC,LastTransitionTime:2021-03-14 00:01:20 +0000 UTC,Reason:KubeletHasSufficientPID,Message:kubelet
has sufficient PID available,},NodeCondition{Type:Ready,Status:True,LastHeartbeatTime:2021-03-14
00:10:24 +0000 UTC,LastTransitionTime:2021-03-14 00:01:30 +0000 UTC,Reason:KubeletReady,Message:kubelet
is posting ready status,},},Addresses:[]NodeAddress{NodeAddress{Type:InternalIP,Address:192.168.0.243,},NodeAddress{Type:Hostname,Address:ham,},},DaemonEndpoints:NodeDaemonEndpoints{KubeletEndpoint:DaemonEndpoint{Port:10250,},},NodeInfo:NodeSystemInfo{MachineID:cf27580027864897b10943e217a56fc8,SystemUUID:34c298ff-1a47-cc43-af38-c8adcfc0329a,BootID:39805f26-8880-4565-8ff2-fbb3bea36977,KernelVersion:4.18.0-240.10.1.el8_3.x86_64,OSImage:CentOS
Linux 8,ContainerRuntimeVersion:containerd://1.4.3,KubeletVersion:v1.20.1,KubeProxyVersion:v1.20.1,OperatingSystem:linux,Architecture:amd64,},Images:[]ContainerImage{ContainerImage{Names:[docker.io/calico/node@sha256:e207db848adb688ae3edac439c650b0ec44c0675efc3b735721f7541e0320100
docker.io/calico/node:v3.18.0],SizeBytes:60502888,},ContainerImage{Names:[k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
k8s.gcr.io/e2e-test-images/agnhost:2.21],SizeBytes:46251468,},ContainerImage{Names:[docker.io/calico/cni@sha256:9d692b3ce9003469f5d50de91dc66ee37d76d78715ed5c1f4884b5d901411489
docker.io/calico/cni:v3.18.0],SizeBytes:44689478,},ContainerImage{Names:[docker.io/calico/pod2daemon-flexvol@sha256:63939cfbd430345c6add6548fc7a22e3082d0738f455ae747dd6264c834bcd4e
docker.io/calico/pod2daemon-flexvol:v3.18.0],SizeBytes:9327966,},ContainerImage{Names:[k8s.gcr.io/pause@sha256:927d98197ec1141a368550822d18fa1c60bdae27b78b0c004f705f548c07814f
k8s.gcr.io/pause:3.2],SizeBytes:299513,},},VolumesInUse:[],VolumesAttached:[]AttachedVolume{},Config:nil,},}
Mar
14 00:12:35.332: INFO: 
Logging kubelet events for node ham
Mar 14 00:12:35.336:
INFO: 
Logging pods the kubelet thinks is on node ham
Mar 14 00:12:35.359:
INFO: calico-node-nmmpz started at 2021-03-14 00:04:02 +0000 UTC (3+1 container
statuses recorded)
Mar 14 00:12:35.359: INFO: \tInit container upgrade-ipam
ready: true, restart count 0
Mar 14 00:12:35.359: INFO: \tInit container
install-cni ready: true, restart count 0
Mar 14 00:12:35.359: INFO: \tInit
container flexvol-driver ready: true, restart count 0
Mar 14 00:12:35.359:
INFO: \tContainer calico-node ready: true, restart count 0
Mar 14 00:12:35.359:
INFO: externalname-service-8l5sf started at 2021-03-14 00:10:11 +0000 UTC
(0+1 container statuses recorded)
Mar 14 00:12:35.359: INFO: \tContainer
externalname-service ready: true, restart count 0
Mar 14 00:12:35.397:
INFO: 
Latency metrics for node ham
Mar 14 00:12:35.397: INFO: 
Logging
node info for node monk
Mar 14 00:12:35.398: INFO: Node Info: &Node{ObjectMeta:{monk
\   73327fee-2f6b-46bc-a142-978d8975030f 1826 0 2021-03-14 00:03:43 +0000
UTC <nil> <nil> map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/os:linux
kubernetes.io/arch:amd64 kubernetes.io/hostname:monk kubernetes.io/os:linux
node-role.kubernetes.io/worker:] map[node.alpha.kubernetes.io/ttl:0 projectcalico.org/IPv4Address:192.168.0.244/24
projectcalico.org/IPv4IPIPTunnelAddr:10.200.102.0 volumes.kubernetes.io/controller-managed-attach-detach:true]
[] []  [{kube-controller-manager Update v1 2021-03-14 00:03:43 +0000 UTC
FieldsV1 {\"f:metadata\":{\"f:annotations\":{\"f:node.alpha.kubernetes.io/ttl\":{}}},\"f:spec\":{\"f:podCIDR\":{},\"f:podCIDRs\":{\".\":{},\"v:\\\"10.200.2.0/24\\\"\":{}}}}}
{kubectl Update v1 2021-03-14 00:03:43 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:labels\":{\"f:node-role.kubernetes.io/worker\":{}}}}}
{kubelet Update v1 2021-03-14 00:04:13 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\".\":{},\"f:volumes.kubernetes.io/controller-managed-attach-detach\":{}},\"f:labels\":{\".\":{},\"f:beta.kubernetes.io/arch\":{},\"f:beta.kubernetes.io/os\":{},\"f:kubernetes.io/arch\":{},\"f:kubernetes.io/hostname\":{},\"f:kubernetes.io/os\":{}}},\"f:status\":{\"f:addresses\":{\".\":{},\"k:{\\\"type\\\":\\\"Hostname\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"InternalIP\\\"}\":{\".\":{},\"f:address\":{},\"f:type\":{}}},\"f:allocatable\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:capacity\":{\".\":{},\"f:cpu\":{},\"f:ephemeral-storage\":{},\"f:hugepages-2Mi\":{},\"f:memory\":{},\"f:pods\":{}},\"f:conditions\":{\".\":{},\"k:{\\\"type\\\":\\\"DiskPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"MemoryPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"PIDPressure\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}},\"k:{\\\"type\\\":\\\"Ready\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}},\"f:daemonEndpoints\":{\"f:kubeletEndpoint\":{\"f:Port\":{}}},\"f:images\":{},\"f:nodeInfo\":{\"f:architecture\":{},\"f:bootID\":{},\"f:containerRuntimeVersion\":{},\"f:kernelVersion\":{},\"f:kubeProxyVersion\":{},\"f:kubeletVersion\":{},\"f:machineID\":{},\"f:operatingSystem\":{},\"f:osImage\":{},\"f:systemUUID\":{}}}}}
{calico-node Update v1 2021-03-14 00:04:22 +0000 UTC FieldsV1 {\"f:metadata\":{\"f:annotations\":{\"f:projectcalico.org/IPv4Address\":{},\"f:projectcalico.org/IPv4IPIPTunnelAddr\":{}}},\"f:status\":{\"f:conditions\":{\"k:{\\\"type\\\":\\\"NetworkUnavailable\\\"}\":{\".\":{},\"f:lastHeartbeatTime\":{},\"f:lastTransitionTime\":{},\"f:message\":{},\"f:reason\":{},\"f:status\":{},\"f:type\":{}}}}}}]},Spec:NodeSpec{PodCIDR:10.200.2.0/24,DoNotUseExternalID:,ProviderID:,Unschedulable:false,Taints:[]Taint{},ConfigSource:nil,PodCIDRs:[10.200.2.0/24],},Status:NodeStatus{Capacity:ResourceList{cpu:
{{2 0} {<nil>} 2 DecimalSI},ephemeral-storage: {{17785946112 0} {<nil>}
16962Mi BinarySI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory: {{8145461248
0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Allocatable:ResourceList{cpu:
{{2 0} {<nil>} 2 DecimalSI},ephemeral-storage: {{16007351475 0} {<nil>}
16007351475 DecimalSI},hugepages-2Mi: {{0 0} {<nil>} 0 DecimalSI},memory:
{{8040603648 0} {<nil>}  BinarySI},pods: {{110 0} {<nil>} 110 DecimalSI},},Phase:,Conditions:[]NodeCondition{NodeCondition{Type:NetworkUnavailable,Status:False,LastHeartbeatTime:2021-03-14
00:04:22 +0000 UTC,LastTransitionTime:2021-03-14 00:04:22 +0000 UTC,Reason:CalicoIsUp,Message:Calico
is running on this node,},NodeCondition{Type:MemoryPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:45 +0000 UTC,LastTransitionTime:2021-03-14 00:03:43 +0000 UTC,Reason:KubeletHasSufficientMemory,Message:kubelet
has sufficient memory available,},NodeCondition{Type:DiskPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:45 +0000 UTC,LastTransitionTime:2021-03-14 00:03:43 +0000 UTC,Reason:KubeletHasNoDiskPressure,Message:kubelet
has no disk pressure,},NodeCondition{Type:PIDPressure,Status:False,LastHeartbeatTime:2021-03-14
00:10:45 +0000 UTC,LastTransitionTime:2021-03-14 00:03:43 +0000 UTC,Reason:KubeletHasSufficientPID,Message:kubelet
has sufficient PID available,},NodeCondition{Type:Ready,Status:True,LastHeartbeatTime:2021-03-14
00:10:45 +0000 UTC,LastTransitionTime:2021-03-14 00:03:43 +0000 UTC,Reason:KubeletReady,Message:kubelet
is posting ready status,},},Addresses:[]NodeAddress{NodeAddress{Type:InternalIP,Address:192.168.0.244,},NodeAddress{Type:Hostname,Address:monk,},},DaemonEndpoints:NodeDaemonEndpoints{KubeletEndpoint:DaemonEndpoint{Port:10250,},},NodeInfo:NodeSystemInfo{MachineID:cf27580027864897b10943e217a56fc8,SystemUUID:ff428e35-79d3-7142-af28-d696ea7e1323,BootID:a90119c6-0adb-4d07-8911-3e5bf7b7162c,KernelVersion:4.18.0-240.10.1.el8_3.x86_64,OSImage:CentOS
Linux 8,ContainerRuntimeVersion:containerd://1.4.3,KubeletVersion:v1.20.1,KubeProxyVersion:v1.20.1,OperatingSystem:linux,Architecture:amd64,},Images:[]ContainerImage{ContainerImage{Names:[docker.io/calico/node@sha256:e207db848adb688ae3edac439c650b0ec44c0675efc3b735721f7541e0320100
docker.io/calico/node:v3.18.0],SizeBytes:60502888,},ContainerImage{Names:[k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
k8s.gcr.io/e2e-test-images/agnhost:2.21],SizeBytes:46251468,},ContainerImage{Names:[docker.io/calico/cni@sha256:9d692b3ce9003469f5d50de91dc66ee37d76d78715ed5c1f4884b5d901411489
docker.io/calico/cni:v3.18.0],SizeBytes:44689478,},ContainerImage{Names:[docker.io/calico/pod2daemon-flexvol@sha256:63939cfbd430345c6add6548fc7a22e3082d0738f455ae747dd6264c834bcd4e
docker.io/calico/pod2daemon-flexvol:v3.18.0],SizeBytes:9327966,},ContainerImage{Names:[k8s.gcr.io/pause@sha256:927d98197ec1141a368550822d18fa1c60bdae27b78b0c004f705f548c07814f
k8s.gcr.io/pause:3.2],SizeBytes:299513,},},VolumesInUse:[],VolumesAttached:[]AttachedVolume{},Config:nil,},}
Mar
14 00:12:35.398: INFO: 
Logging kubelet events for node monk
Mar 14 00:12:35.400:
INFO: 
Logging pods the kubelet thinks is on node monk
Mar 14 00:12:35.420:
INFO: execpodq6fvf started at 2021-03-14 00:10:23 +0000 UTC (0+1 container
statuses recorded)
Mar 14 00:12:35.420: INFO: \tContainer agnhost-container
ready: true, restart count 0
Mar 14 00:12:35.420: INFO: calico-node-jg24p
started at 2021-03-14 00:04:03 +0000 UTC (3+1 container statuses recorded)
Mar
14 00:12:35.420: INFO: \tInit container upgrade-ipam ready: true, restart
count 0
Mar 14 00:12:35.420: INFO: \tInit container install-cni ready:
true, restart count 0
Mar 14 00:12:35.420: INFO: \tInit container flexvol-driver
ready: true, restart count 0
Mar 14 00:12:35.420: INFO: \tContainer calico-node
ready: true, restart count 0
Mar 14 00:12:35.440: INFO: 
Latency metrics
for node monk
Mar 14 00:12:35.440: INFO: Waiting up to 3m0s for all (but
0) nodes to be ready
STEP: Destroying namespace \"services-6203\" for this
suite.
[AfterEach] [sig-network] Services
  /workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/service.go:749
"
